pom.xml

```diff
@@ -44,6 +44,8 @@
   <properties>
     <hadoop.version>2.6.0-cdh5.5.0</hadoop.version>
     <spark.version>1.5.0-cdh5.5.0</spark.version>
+    <hadoop.scope>compile</hadoop.scope>
+    <spark.scope>compile</spark.scope>
```

Comment:
I didn't ask you to remove the hadoop override, just the Spark one, because as our previous conversation shows, it is not needed. There are no Spark jars in the final package.